packages/core/src/lib/util/nested.util.ts

```diff
@@ -93,6 +93,25 @@ export function getNestedSlug(
   return '';
 }
 
+export function isNodeEditable(collection: Collection, node: SingleTreeNodeData | TreeNodeData): boolean {
```

Comment:
Really silly question: What is the order in this file? Seems like the new functions are just randomly in the middle.

Reply:
I'm not aware of any fixed order (like alphabetical).

Tried to place the added functions where they fit best, based on what they and the adjacent functions do.